orchestrator/register_delegate_keys/src/main.rs

```diff
@@ -127,6 +127,7 @@ async fn main() {
         cosmos_granter,
         ethereum_wallet,
         (0f64, "".to_string()),
+        500_000,
```

Comment:
is that a reasonable default gas value?

Reply:
it was the previously default gas value. 

I think it may be a bit high for the delete_keys txs (may need to collect metrics) but since this operation is not likely to be done frequently (only one time per validator for initialisation) keeping it that way is probably okay